### DISCUSSION TITLE
Bearware.dk logins are bye bye

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#1800] [General]
Hello,

Since about half an hour now, it is not possible to authenticate to a TeamTalk server using bearware.dk web logins.
All attempts are rejected.
If restarting a server, we have to re-authenticate in loop and for running servers, it looks like impossible to authenticate using this method.

**Comment by bear101** [comment]
Phew, it looks like my website host changed som security setting. When the TeamTalk Pro server contacts www.bearware.dk the website returns:
<h1>406 Security Incident Detected</h1>
<p>Your request was blocked.</p>

<h3>If you are the owner of the website:</h3>
<p>The Website Application Firewall that is protecting your website has blocked this request for being suspicious.</p>
<p>You can see the detailed reason for this in your webserver logs.</p>

<h3>If you are the visitor:</h3>
<p>The public IP address assigned to you, by your internet provider, might be suffering from poor reputation: <a href='https://www.simply.com/rbl/?ip=xx.xx.xx.xx'>Look up IP reputation here</a>. IP addresses from VPN providers or public networks often have poor reputation.</p>

**Comment by bear101** [comment]
If you comment out the "bearware" account in tt5prosrv.xml then the initial BearWare.dk login will be skipped. Still it will not be possible to log in using BearWare.dk WebLogin until I fix the problem

**Comment by bear101** [comment]
Should be working again

**Reply by Nardol** [reply]
Confirmed, thanks! :+1:
